### PR TITLE
Load GS_Model using custom config.yaml file

### PR DESCRIPTION
**Pull request by MACILLAS**
Streamlining the way we configure the gaussian splat model by specifying the config.yaml file in the .ply directory.

### CODE REVIEW

File: camera_pos_utils.py [hunk 1]
```diff
@@ -138,5 +138,5 @@ def translate4(x, y, z):
                       [-0.68636268, 0.42995355, -0.58655453]])
     T_vec = np.array([-0.32326042, -3.65895232, 2.27446875])
     init_pose = compose_44(R_mat, T_vec)
-    imagelocs = Images_Meta("/home/cviss/PycharmProjects/GS_Stream/data/UW_tower/sparse/0/images.txt")
+    imagelocs = ImagesMeta("/home/cviss/PycharmProjects/GS_Stream/data/UW_tower/sparse/0/images.txt")
```
Comment (KyungwanHan) [inline camera_pos_utils.py:141]: Just out of curiosity, what is images.txt? What's the role of this file?

File: render_wrapper.py [hunk 2]
```diff
@@ -68,22 +74,18 @@ def __init__(self,
         self.pipeline = DummyPipeline()
 
         self.gaussians = GaussianModel(3)  # 3 is the default sh-degree
-        self.gaussians.load_ply(ply_path)
+        self.gaussians.load_ply(config["ply_path"])
```
Comment (KyungwanHan) [inline render_wrapper.py:77]: Does that require two configuration file paths, config_path and ply_path? Are there any other configuration files? And what does 'ply' mean?

**Review by KyungwanHan — APPROVED** [review]
LGTM. I just left some questions.

File: camera_pos_utils.py (hunk 1 above)
Comment (KyungwanHan) [reply camera_pos_utils.py:141]: As far as I know, class names in Python should follow the CamelCase convention. I think changing it was a good move. 👍

Comment (MACILLAS) [reply camera_pos_utils.py:141]: Thanks! Yeah I was super sloppy with the naming conventions... The images.txt is from colmap and it contains the image name and camera poses (as quaternion). This is necessary for me to get the closest images to the current pose.

File: render_wrapper.py (hunk 2 above)
Comment (MACILLAS) [reply render_wrapper.py:77]: To initialize the GS_Model class you would only need the config_path... The config.yaml contains the field ply_path, which points to the gaussian splat model (PLY file) relative to the project directory.

**Comment by KyungwanHan** [comment]
Thank you for sharing your answers!

**Pull request merged**